src/slack.ts

```diff
@@ -118,7 +124,15 @@ async function send(
   for (const [step, status] of Object.entries(jobSteps)) {
     checks.push(`${stepIcon(status.outcome)} ${step}`)
   }
-  const fields = []
+
+  if (commits.length) {
+    fields.push({
+      title: '',
```

Comment:
If we modify the push event title like commented above, we could assign 'Commits' as title for these fields too which would make the whole thing more uniform!